recipes/xrftomo/meta.yaml

```diff
@@ -0,0 +1,60 @@
+
+{% set name = "XRFtomo" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }} 
+  version: {{ version }}
+
+source:
+  url: https://github.com/tomography/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 986c4ea85c7fe8bcef274b7085209b74e8a93acd444ad1f9b947fb3f1352cbbb
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - dxchange
+    - dxfile
+    - h5py
+    - matplotlib
```

Comment:
I assume you need the `Qt` backend with `matplotlib`?  If you only need to render images to files, then `matplotlib-base` can be used.

Reply:
The first statement is correct.